runtime/src/bank.rs

```diff
@@ -4961,7 +4956,6 @@ impl Bank {
                 enable_return_data_recording: true,
             },
             &mut ExecuteTimings::default(),
-            Some(1000 * 1000),
```

Comment:
Generally, I like this change as we have lots of bloated argument lists in the codebase. That being said, this one looks like it might be a slight change of behavior.
- Currently, callers of this function will always get a value of `Some(1e6)`
- With your change, the value will be whatever is initialized in the first `Bank` (ie CLI arg)

It looks like we have this caller here; I'm not super familiar with BanksServer so will need to dig a little (or let someone else chime in) to ensure we're good with the change:
https://github.com/anza-xyz/agave/blob/b35e17cb0c4e7b8f98f1fa3a932cd90dd8bff644/banks-server/src/banks_server.rs#L371